phantom_connector.py

```diff
@@ -534,60 +559,82 @@ def _find_artifacts(self, param):
         if limit_search:
             container_ids = list(
-                set([
-                    a for a in [
-                        int(z) if isinstance(z, int) or z.isdigit() else None for z in [
-                            self.get_container_id() if y == "current" else y for y in
-                            [x.strip() for x in container_ids.replace(",", " ").split()]
+                set(
+                    [
+                        a
+                        for a in [
+                            int(z) if isinstance(z, int) or z.isdigit() else None
+                            for z in [
+                                self.get_container_id() if y == "current" else y
+                                for y in [x.strip() for x in container_ids.replace(",", " ").split()]
+                            ]
                         ]
-                    ] if a
-                ])
+                        if a
+                    ]
+                )
             )
             action_result.update_param({"container_ids": str(sorted(container_ids)).strip("[]")})
 
         if limit_search and not container_ids:
-            action_result.update_summary({'artifacts_found': 0, 'server': self._base_uri})
+            action_result.update_summary({"artifacts_found": 0, "server": self._base_uri})
             return action_result.set_status(phantom.APP_SUCCESS)
 
         cef_key = param.get("cef_key")
 
-        exact_match = param.get('exact_match', False)
+        exact_match = param.get("exact_match", False)
 
         if exact_match and not cef_key:
             values = '"{}"'.format(values)
 
-        url_enc_values = quote(values, safe='')
+        url_enc_values = quote(values, safe="")
 
         if cef_key and exact_match:
-            endpoint = '/rest/artifact?_filter_cef__{}={}&page_size=0&pretty'.format(quote(cef_key, safe=''), repr(url_enc_values))
+            endpoint = f"/rest/artifact?_filter_cef__{quote(cef_key, safe='')}={repr(url_enc_values)}&pretty"
         elif cef_key:
-            endpoint = '/rest/artifact?_filter_cef__{}__{}={}&page_size=0&pretty'.format(quote(cef_key, safe=''),
-                                                                                        "icontains", repr(url_enc_values))
+            endpoint = f"/rest/artifact?_filter_cef__{quote(cef_key, safe='')}__icontains={repr(url_enc_values)}&pretty"
         else:
-            endpoint = '/rest/artifact?_filter_cef__{}={}&page_size=0&pretty'.format("icontains", repr(url_enc_values))
+            endpoint = f"/rest/artifact?_filter_cef__icontains={repr(url_enc_values)}&pretty"
 
         if limit_search:
-            endpoint += '&_filter_container__in={}'.format(container_ids)
+            endpoint += "&_filter_container__in={}".format(container_ids)
 
-        ret_val, response, resp_data = self._make_rest_call(endpoint, action_result)
+        records = list()
```

Comment:
```suggestion
        records = []
```

Reply:
Thanks for review comment, based on the priority - I am merging this PR, as changing the code will require testing and other efforts. Also this function is not that impacting the code.